src/resolversFactory.js

```diff
@@ -1,7 +1,18 @@
 var util = require('util');
 var http = require('http');
+var peliasConfig = require( 'pelias-config' ).generate();
+
+
+function createWofPipResolver(url, config) {
+  config = config || peliasConfig;
+
+  var maxConcurrentReqs = 1;
+  if (config.imports.adminLookup && config.imports.adminLookup.maxConcurrentReqs) {
+    maxConcurrentReqs = config.imports.adminLookup.maxConcurrentReqs;
+  }
+
+  http.globalAgent.maxSockets = maxConcurrentReqs;
```

Comment:
I'm not proposing anything that changes the interface to this module, but instead to avoid changing settings on `http.globalAgent` which, as the name implies, is basically a global variable of default settings for all HTTP requests in the entire node process. Instead it's possible to create an agent variable in local scope and pass it to `http.get` instead. See https://github.com/pelias/fuzzy-tester/blob/master/lib/request_urls.js#L31 for how it works